packages/taco/schema-docs/condition-schemas.md

```diff
@@ -6,6 +6,7 @@ _Union of the following possible types:_
 
 - [RpcCondition](#rpccondition)
 - [TimeCondition](#timecondition)
+- [AddressAllowlistCondition](#walletallowlistcondition)
```

Comment:
Is this link correct? It still has `[wallet]allowlistcondition` instead of `[address]allowlistcondition`.